### ISSUE TITLE
Does it support ws+nginx mode?

### ISSUE THREAD

**Issue by flowercow** [zw963/asuswrt-merlin-v2ray#6]
i changed the config file and found that domain can not be solved.
Does that dns query not supported over ws+tls+nginx mode?

**Comment by flowercow** [comment]
after apply iptables rules, i can't accese my router through openvpn. I think something may be wrong with udp settings.

**Comment by flowercow** [comment]
When I use patch_router, it seems to work. But download speed is affected, down from 230M/s to 30M/s, when i use some video apps,  there is about 5 seconds lag. 
maybe dns query over ws+tls cause that?

**Comment by zw963** [comment]
>  And I can't accese my router through openvpn. I think something may be wrong with udp settings.

Yes, maybe, please help on improve this, it just a personal script anyway.



> But after I apply dnsmasq config( 127.0.0.1:65053 as dns name server, ) and iptable rules, dns query is very slow and some domain can not be solved.

I rarely meet a situation where DNS queries are very slow, though, not so quickly, but it enough for me, most of time, i search  on google and see 720p youtube.

> When I use patch_router, it seems to work. But download speed is affected, down from 230M/s to 30M/s, when i use some video apps, there is about 5 seconds lag.
> maybe dns query over ws+tls cause that?

Wow, What is your's vps, 30M/s from proxy???

if this speed is not from proxy, that maybe ip/domain filter issue, you need add your own into white list, can you please try deploy with the newest version again?

if this issue is from proxy, there may exist many reason which cause speed issue, following is some:

1.  you router is too old, run v2ray is overload for you router.
2.  vps is overload
3.  network to your's vps.
4.  reboot router every night (automatically) to ensure not run v2ray too long.

**Comment by zw963** [comment]
>  maybe dns query over ws+tls cause that?

I never use this mode, though i can, you can try reproduce with newest script without use ws+tls, and try again.

i suspect you meet white list issue, that is, router select use proxy to visit china site.
